### PR TITLE
Create <Alert /> Component

### PR DESCRIPTION
**Pull request by davidenwang**
Added alerts with prop "variant" which can be "success", "warning", etc...

image

Resolves: #16

### CODE REVIEW

**Comment by tko22** [comment]
I was thinking we had <Alert.Icon/> and <Alert.Link/> but im not sure whether we need that - seems like ur implementation works well

I guess we'll need to document that you need to put an svg to include an icon

File: src/components/Alert/Alert.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  // align
+  vertical-align: middle;
+  align-items: center;
+
+  // size
+  padding: 0px 18px;
+  height: 55px;
+
+  svg {
+    margin-right: 18px;
```
Comment (tko22) [inline src/components/Alert/Alert.js:30]: should we include `float: left` or any margin on the left??

Comment (davidenwang) [reply src/components/Alert/Alert.js:30]: we could do float left, but the padding should push the icon to the right by the appropriate amount

File: src/theme.js [hunk 2]
```diff
@@ -145,8 +145,8 @@ const tableFormAlertStyle = {
   fontFamily: fonts.main,
   fontWeight: fontWeights.regular,
   fontSize: '18px',
-  letterSpacing: letterSpacings.third, // 0.3px
-  lineHeight: '24px',
+  letterSpacing: letterSpacings.third, // 0.3[x]
+  lineHeight: '21px',
```
[the remark this answers is not in the document]
Comment (davidenwang) [reply src/theme.js:149]: sorry I saw in the figma that line height for alerts was 21px so I changed it. It seems to be similar for form and table. I couldn't find 24px but I might have not looked hard enough. I'll merge for now but lemme know if it should be changed I can tweak it.

**Comment by davidenwang** [comment]
> I was thinking we had <Alert.Icon/> and <Alert.Link/> but im not sure whether we need that - seems like ur implementation works well
> 
> I guess we'll need to document that you need to put an svg to include an icon

For sure I just followed https://primer.style/components/Flash, but if you prefer the Icon and Link we can change in another PR

**Pull request merged**